### ISSUE TITLE
installing on debian 8.8

### ISSUE THREAD

**Issue by piahoo** [presslabs/gitfs#269]
Hi
I tried to install gitfs on debian, but without a luck

error is:

  File "/usr/lib/python2.7/dist-packages/cffi/ffiplatform.py", line 51, in _build
    raise VerificationError('%s: %s' % (e.__class__.__name__, e))
cffi.ffiplatform.VerificationError: CompileError: command 'x86_64-linux-gnu-gcc' failed with exit status 1

any hints?

**Comment by narcisgarcia** [comment]
To compile/package gitFS is needed libgit2-dev (>= 0.24)
I could do this with Debian 9 as root:
```
apt install build-essential checkinstall automake python-virtualenv libffi-dev python3-dev libgit2-dev
wget https://github.com/PressLabs/gitfs/archive/master.zip
unzip master.zip
rm master.zip
mv gitfs-master gitfs
cd gitfs
# Fix subversion reference for virtualenv:
sed -i -e 's|python3.4|python3|g' Makefile
```
Write a "gitfs.spec" file with a line:
`Release: 1`
Write a "description-pak" file with a line:
`Version controlled file system`
Continue to compile and package:
```
make
checkinstall -D --fstrans --default --reset-uids "--maintainer=MyName" --pakdir=.. --pkgname=gitfs --requires=python3-minimal,python-fusepy,python-pygit2,python-setuptools --pkgrelease=1 --install=no make install
cd ..
rm -r gitfs
```
Package dependencies should be fixed about version of python-pygit2 (>= 0.24) because gitfs requires this version at least.
Anyway, you will need to backport dependencies not available at plain Debian 8 repositories.

**Comment by vtemian** [comment]
We've just released 0.5.1 with updated dependencies and fresh new debs.
Can you please try again, with the newer version?

Sorry for the delay.
